pkgs/development/compilers/rust/rustc.nix

```diff
@@ -133,6 +136,11 @@ in stdenv.mkDerivation rec {
 
     # Useful debugging parameter
     # export VERBOSE=1
+  '' + lib.optionalString (stdenv.isDarwin && stdenv.isx86_64) ''
+    # jemalloc's configure script misdetects the virtual address size on Darwin under Rosetta 2.
```

Comment:
Is it disingenuous to say this, since it's really a Rosetta issue? Or is it more clear/simple to just say the issue rather than the root cause?

(Apologies for all the nits.)

Reply:
dropped the comment, kept the link to the issue

Reply:
Sure, that works too.